vwifi.c

```diff
@@ -53,38 +58,76 @@ static inline struct owl_ndev_priv_context *ndev_get_owl_context(
     return (struct owl_ndev_priv_context *) netdev_priv(ndev);
 }
 
-/* Helper function that will prepare structure with "dummy" BSS information and
- * "inform" the kernel about "new" BSS
- */
-static void inform_dummy_bss(struct owl_context *owl)
+static s32 rand_int(s32 low, s32 up)
 {
-    struct cfg80211_bss *bss = NULL;
-    struct cfg80211_inform_bss data = {
-        /* the only channel */
-        .chan = &owl->wiphy->bands[NL80211_BAND_2GHZ]->channels[0],
-
-        /* signal "type" not specified in this module, so it is unused.
-         * It can be some kind of percentage from 0 to 100 or mBm value.
-         * signal "type" may be specified before wiphy registration by setting
-         * wiphy->signal_type
-         */
-        .scan_width = NL80211_BSS_CHAN_WIDTH_20,
-        .signal = 1337,
-    };
+    s32 result = (s32) get_random_u32();
+    result %= (up - low + 1);
+    result = abs(result);
+    result += low;
+    return result;
+}
 
-    /* array of tags that retrieved from beacon frame or probe responce */
-    char ie[SSID_DUMMY_SIZE + 2] = {WLAN_EID_SSID, SSID_DUMMY_SIZE};
-    memcpy(ie + 2, SSID_DUMMY, SSID_DUMMY_SIZE);
+/* Helper function for generating BSSID from SSID */
+static void generate_bssid_from_ssid(u8 *result, const char *ssid)
```

Comment:
It should be `generate_bssid_with_ssid`.

Reply:
fixed on https://github.com/sysprog21/vwifi/pull/4/commits/0c6c0135e25ccf4e0a194724debba5862c1d8941